plugin/src/main/java/com/denizenscript/denizen/utilities/AdvancedTextImpl.java

```diff
@@ -105,4 +106,16 @@ public InventoryView openAnvil(Player player, Location loc) {
     public void teleportPlayerRelative(Player player, Location loc) {
         player.teleport(loc, PlayerTeleportEvent.TeleportCause.PLUGIN);
     }
+
+    public void registerBrewingRecipe(String keyName, ItemStack result, ItemStack[] inputItem, boolean inputExact, ItemStack[] ingredientItem, boolean ingredientExact){
+        throw new UnsupportedOperationException();
+    }
+
+    public void clearBrewingRecipes(){
+        throw new UnsupportedOperationException();

```

Comment:
Please test on a Spigot server - this exception is going to fire and crash reloads.

Reply:
Ah, I left that in on accident, my intention was to just replace that with a return statement, or is there a better alternative to put there?

Reply:
just leave the method empty

Reply:
Gotcha